src/compiler/cpp_generator.cc

```diff
@@ -54,6 +54,16 @@ std::string FilenameIdentifier(const std::string& filename) {
   }
   return result;
 }
+
+std::string EscapeVariableDelimiters(std::string&& mut_str) {
+  size_t index = 0;
```

Comment:
Consider instead using `absl::StrReplaceAll()` for this.

Reply:
That was my first instinct, but the `grpc_plugin_support` target has very few dependencies, and I'm not entirely sure why, it seems almost by design. Maybe for code bloat reasons? Anyhow, that's the only reason I didn't use abseil here.

Reply:
I feel like we should be careful on adding more dependency to `grpc_plugin_support`.